lib/oauth2/access_token.rb

```diff
@@ -3,6 +3,7 @@ class AccessToken
     attr_reader :client, :token, :expires_in, :expires_at, :params
     attr_accessor :options, :refresh_token
 
+    # Should these methods be deprecated?
```

Comment:
I'm not sure what we want to do with these. We could remove them. The `from_kvform` in particular isn't used at all. Seems to be a convenience method. if you're handling token's yourself. I'd be all for dropping them in 2.x  and making this class simpler and general purpose with no assumptions about how the access_token is provided in the hash.

Reply:
I use it to store/restore access token. You could mark it deprecated after providing another better solution, otherwise, leaving a comment like this will confuse other. Thanks.

Reply:
Yup the comment was to draw attention to it for the purpose of this review. I'm happy to just remove this comment if we think these methods stay. It still feels like these methods belong elsewhere.

Reply:
Agreed on all counts.